recipes/openai_textcat_multilabel.py

```diff
@@ -0,0 +1,527 @@
+import copy
```

Comment:
Is there a reason why this had to happen in a separate file? The Prodigy [`textcat.correct`](https://prodi.gy/docs/recipes/#textcat-correct) recipe will adapt based on value for `--labels`. Is there a reason that cannot happen here?